src/modules/print/BasicsPrintView.js

```diff
@@ -698,6 +706,8 @@ class BasicsPrintView extends PrintView {
         return {
           color: this.colors[type],
           count: _.get(data, `${day.date}.total`, _.get(data, `${day.date}.count`, 0)),
+          // translation missing
```

Comment:
```suggestion
          // FIXME translation missing
```
Just to identify this comment purpose.